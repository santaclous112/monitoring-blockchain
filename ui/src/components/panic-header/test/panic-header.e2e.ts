import { newE2EPage } from '@stencil/core/testing';

describe('panic-header', () => {
  it('renders', async () => {
    const page = await newE2EPage();
    await page.setContent('<panic-header></panic-header>');

    const element = await page.find('panic-header');
    expect(element).toMatchSnapshot();
  });
});
