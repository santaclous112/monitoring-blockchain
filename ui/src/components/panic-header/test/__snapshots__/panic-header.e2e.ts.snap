// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`panic-header renders 1`] = `
<panic-header>
  <svc-header imgpath="/assets/logos/panic_logo.png" imgposition="start" imgurl="/" menuposition="end">
    <div class="panic-header__menu" slot="menu">
      <svc-dropdown-menu label="Networks"></svc-dropdown-menu>
      <svc-dropdown-menu label="Settings"></svc-dropdown-menu>
    </div>
  </svc-header>
</panic-header>
`;
